.github/workflows/build-master.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will build a package using Gradle.
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Build master branch
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Java
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build with Gradle
+      run: gradle build
+    - name: Generate sources jar with Gradle
+      run: gradle sourcesjar
+    - name: Generate JavaDoc jar with Gradle
+      run: gradle javadocjar
+    - name: Upload package as artifact
```

Comment:
These are built with every commit. Do we need to upload the package as an artifact for each commit?

Reply:
This is only done for commits to the master branch.